Sources/PrimerSDK/Classes/Core/3DS/Data Models/3DS.swift

```diff
@@ -320,22 +324,34 @@ public class ThreeDS {
         init(from decoder: Decoder) throws {
             let container = try decoder.container(keyedBy: CodingKeys.self)
 
-            if let threeDSDeclinedAPIResponse = (try? container.decode(ThreeDS.DeclinedAPIResponse.self, forKey: .authentication)) {
+            if let threeDSDeclinedAPIResponse =
```

Comment:
The readability of these is worse now IMO - can we make the line length longer for this file / function?

Reply:
That is something we should discuss. My aim was to follow default SwiftLint rules. If we alter the rules to silence them in our DebugApp, no one can guarantee that the merchant will follow the same rules... WDYT?

Reply:
What about now? @jnewc